radixdlt-java/src/test/java/com/radixdlt/client/core/crypto/RadixECKeyPairsTest.java

```diff
@@ -0,0 +1,50 @@
+package com.radixdlt.client.core.crypto;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.hamcrest.core.IsNot.not;
+import static org.junit.Assert.assertThat;
+
+public class RadixECKeyPairsTest {
+
+    @BeforeClass
+    public static void beforeSuite() {
+        // Ensure the BouncyCastle providers are loaded into memory
+        // (because BouncyCastle SHA-256 is used in "seed" tests).
+        ECKeyPairGenerator.newInstance();
```

Comment:
Is this necessary?

Reply:
No, not `newInstance()`, but `install()` is necessary sometimes, depending on what has been run and what hasn't before these tests are executed.

Reply:
Hm... the code comment isn't too clear about the fact that the "seededKeyPair" implementation will try to retrieve the "BC" provider and it's "SHA-256" implementation. Unless we ensure that said provider exists in the java `Security` repository (by `install()`-ing it), we will get an exception and thereby failing tests.

On top of all, this only happens if no `ECKeyPairGenerator` related code has been run before these tests are executed - "random" at first glance.